PRIVACY.md

```diff
@@ -1,134 +1,166 @@
 
-SimpleX Chat is built on top of SimpleX messaging and application platform that uses a new message routing protocol allowing to establish private connections without having any kind of addresses that identify its users - we don't use emails, phone numbers, usernames, identity keys or any other user identifiers to pass messages between the users.
+While SimpleX Chat Ltd is not a communication service provider, and provide any preset relays "as is", as experimental, without any guarantees of availability or data retention, we are committed to maintain a high level of availability, reliability and security of these preset relays. We will be adding alternative preset infrastructure providers to the software in the near future, and you will continue to be able to use any other providers or your own servers.
 
-SimpleX Chat security assessment was done in October 2022 by [Trail of Bits](https://www.trailofbits.com/about), and most fixes were released in v4.2.0 – see [the announcement](./blog/20221108-simplex-chat-v4.2-security-audit-new-website.md).
+We see users and data sovereignty, and device and provider portability as critically important properties for privacy and security of the communications.
 
-### Information you provide
+SimpleX Chat security assessment was done in October 2022 by [Trail of Bits](https://www.trailofbits.com/about), and most fixes were released in v4.2 – see [the announcement](./blog/20221108-simplex-chat-v4.2-security-audit-new-website.md).
+
+### Your information
 
 #### User profiles
 
-We do not store user profiles. The profile you create in the app is local to your device.
+Servers used by SimpleX Chat apps do not create, store or identify user profiles. The profiles you can create in the app are local to your device, and can be removed at any time.
+
+When you create the local profile, no records are created on any of the relay servers, and infrastructure providers, whether SimpleX Chat Ltd or any other, have no access to any part of your information, and even to the fact that you created a profile - it is a local record stored only on your device. That means that if you delete the app, and have no backup, you will permanently lose all your data and the private connections you created with other users.
 
-When you create a user profile, no records are created on our servers, and we have no access to any part of your profile information, and even to the fact that you created a profile - it is a local record stored only on your device. That means that if you delete the app, and have no backup, you will permanently lose all the data and the private connections you create with other users.
+You can transfer the profile to another device by creating a backup of the app data and restoring it on the new device, but you cannot use to devices with the copy of the same profile at the same time.
```

Comment:
```suggestion
You can transfer the profile to another device by creating a backup of the app data and restoring it on the new device, but you cannot use two devices with the copy of the same profile at the same time.
```

Reply:
or better "multiple" or "more than one"

Reply:
My concern is that technically you can use the profile. the database on multiple devices bit you will face issues on your devices quite soon, some messages will not be delivered etc. So it is not that you can not at all. but it is not going to function as intended andvarious unexpected errors you will face and will lose some of your new messages etc. It is or an intended usage form, not recommended and after key rotations it will fail too. 

I think all this should be made clear that if the user clones its database onto more than one device, they will still open and seemingly work for a very limited time etc. For example in Session chat app, the biggest concern is the fact that there you can clone accounts (IDs by the seed) onto an unlimited number of devices and it still works "normally" and no one will ever notice it either.